docs/tracing/create-tracing-app.md

```diff
@@ -0,0 +1,254 @@
+
+Name the Service:
+
+- Enter a meaningful name for your application service
+- This should match your actual service name in your application
+
+Follow SDK Instructions:
+The UI will provide complete, language-specific instructions including:
+
+- Required dependencies and installation commands
+- Code examples for auto-instrumentation (recommended)
+- Manual instrumentation examples (advanced)
+- Environment variables and configuration options
+- Framework-specific guidance
+
+Add Multiple Services (Optional):
+
+- Click "Add Another Service" to configure multiple services at once
+- Each service will have its own configuration and will appear in the service list
+
+### Step 3: Install and Configure Sematext Agent
+
+The wizard continues with agent installation options:
+
+![Agent Installation Options](/docs/images/tracing/tracing-new-app-creation-4a.png)
+
+3.1 Choose Deployment Method:
+
+Select from four deployment methods:
+
+Single Host:
+
+- Best for: Development environments, single servers, or testing
+- Platforms: Linux (various distributions), Windows  
+- Installation: GUI installer (Windows) or command-line installation (Windows / Linux)
+
+Server Fleet:
+
+- Best for: Production environments with multiple servers managed centrally
+- Management: Ansible, Windows Group Policy
+- Approach: Configuration management for multiple hosts
+
+Containerized:
+
+- Best for: Docker environments, container orchestration platforms
+- Platforms: Docker, Swarm, Nomad, AWS ECS
+- Deployment: As a container in your containerized environment
+
+Kubernetes:
+
+- Best for: Kubernetes clusters (AKS, EKS, GKE, OpenShift, Rancher)
+- Methods: Helm, kubectl, Sematext Operator
+- Deployment: As a DaemonSet across your Kubernetes cluster
+
+3.2 Follow Installation Instructions:
+
+The UI provides step-by-step installation commands for your chosen method:
+
+Agent Configuration:
+The agent will be automatically configured with:
+
+- Your Infra App token for authentication
+- OTLP receiver enabled for trace collection  
+- Your Traces token for sending data to Sematext Cloud
+- Service names you configured in Step 2
+- Default ports: 4337 (gRPC), 4338 (HTTP)
+
+Custom Port Configuration (Optional):
+
+- Modify default OTLP ports if needed
+
+### Step 4: Complete Setup
+
+4.1 Verify Installation:
+
+- The wizard will show "Host Detected" when the agent is running correctly
+- Services will appear as "Healthy" when properly configured
+
+4.2 Generate Test Traffic:
+
+- Make requests to your instrumented application
+- Traces should appear in the newly created Tracing App within seconds
+
+## What You'll Have After Completion
+
+### Tokens and Configuration
+
+- Infra App Token: For agent authentication
+- Traces Token: For sending trace data to Sematext Cloud
+- Service Configuration: All configured services ready to send traces
+
+### Ready-to-Use Features
+
+- Tracing Overview: Key metrics and service health
```

Comment:
Do you think any of the individual components will require an explanation? As in to help people understand what they represent, what they are telling them about their applications, how to interpret them and such.

Reply:
We have dedicated pages to each one of them, added links.